conformance/README.md

```diff
@@ -31,10 +31,9 @@ preload-nginx-container        Preload NGINX container on configured kind cluste
 prepare-nkg-dependencies       Install NKG dependencies on configured kind cluster
 reset-go-modules               Reset the go modules changes
 run-conformance-tests          Run conformance tests
-undo-image-update              Undo the NKG image name and tag in deployment manifest
-uninstall-nkg                  Uninstall NKG on configured kind cluster
+undo-manifests-update          Undo the changes in the manifest files
+uninstall-nkg                  Uninstall NKG on configured kind cluster and undo manifest changes
 update-go-modules              Update the gateway-api go modules to latest main version
-update-nkg-manifest            Update the NKG deployment manifest image name and imagePullPolicy
```

Comment:
why did we remove it from here? we still reference it in this README

Reply:
It was accidental - I removed the comment so it didn't render when I ran `make` 😬